lib/stat_tracker.rb

```diff
@@ -142,5 +142,59 @@ def average_goals_by_season
     avg_season_goals
   end
 
-  #league stats
+  def count_of_teams
+    team_count = 0
+    contents = CSV.open(@team_path, headers: true, header_converters: :symbol)
+    contents.each do |row|
```

Comment:
may need to research this later but im sure we can count the teams by using a CSV method that counts the lines excluding the headers. just a guess tho idk